thirdeye/thirdeye-frontend/app/utils/anomalies-tree-parser.js

```diff
@@ -0,0 +1,563 @@
+import { isEmpty } from "@ember/utils";
+import { set } from "@ember/object";
+import moment from "moment";
+
+const CLASSIFICATIONS = {
+  METRICS: {
+    KEY: "metrics",
+    COMPONENT_PATH: "entity-metrics",
+    DEFAULT_TITLE: "Metric Anomalies"
+  },
+  GROUPS: {
+    KEY: "groups",
+    COMPONENT_PATH: "entity-groups",
+    DEFAULT_TITLE: "ENTITY:"
+  },
+  ENTITIES: {
+    KEY: "entities",
+    COMPONENT_PATH: "parent-anomalies",
+    DEFAULT_TITLE: "Entity"
+  }
+};
+const BREADCRUMB_TIME_DISPLAY_FORMAT = "MMM D HH:mm";
```

Comment:
I suggest you put this time format in the constant.js or to create a constant folder. it's a common useful variable. We can inject into the function.

Reply:
Sure.